### PR TITLE
Dropout=0.1 on 5L/256d + T_max=50 (regularization for wall-shear)

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

**Dropout=0.1 on 5L/256d + T_max=50 — regularization as wall-shear generalization lever.**

The current best (alphonse PR #174, val_abupt=6.9549%) uses no dropout (`--model-dropout 0.0` default). The wall-shear axes (wsy=8.73%, wsz=10.58%) are 5–7pp above their AB-UPT targets, which is a strong signal of insufficient generalization on high-frequency surface gradients. Light dropout (0.1) is a classical remedy:

- Transformer models often overfit to training-set geometry without dropout, particularly on sparse/noisy fields like wall-shear which have high spatial variability.
- Dropout=0.1 in vision/physics transformers (ViT, FNO variants) consistently improves val metrics when train/val gap is large on fine-grained spatial quantities.
- No dropout experiments have been run on the 5L/256d architecture — this is a blind spot in the current search.

No code changes required — `--model-dropout` is an existing CLI flag.

## Instructions

Run **one trial** on the current best architecture with `--model-dropout 0.1`. No code changes needed.

```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --model-dropout 0.1 \
  --no-compile-model \
  --agent gilbert \
  --wandb-group bengio-wave15 \
  --wandb-name gilbert-dropout01-5l256d-tmax50
```

ACK this PR within 30 minutes by posting your W&B run ID as a comment. Then post metrics at each gate.

## Gates (kill if NOT below)

Kill thresholds — pass these via `--kill-thresholds` flag:

```
--kill-thresholds "89082:val_primary/abupt_axis_mean_rel_l2_pct<13.0,178164:val_primary/abupt_axis_mean_rel_l2_pct<11.0,267246:val_primary/abupt_axis_mean_rel_l2_pct<9.5,534492:val_primary/abupt_axis_mean_rel_l2_pct<7.8"
```

| Epoch | Step    | abupt gate | Notes |
|-------|---------|------------|-------|
| ep5   | 89,082  | < 13.0%    | Sanity check |
| ep10  | 178,164 | < 11.0%    | Trajectory check |
| ep15  | 267,246 | < 9.5%     | Clear descent required |
| ep30  | 534,492 | < 7.8%     | Must be approaching baseline |
| ep50  | final   | < 6.9549%  | Must beat current best |

At each gate, post a comment with the full per-axis table: abupt, sp, vp, wsx, wsy, wsz.

## Reporting

Post a final comment with:
1. Per-channel val metrics at ep5/10/15/30/50 for all axes (abupt, sp, vp, wsx, wsy, wsz)
2. Best-checkpoint test_primary metrics for all 5 axes
3. Best epoch and W&B run ID
4. Comparison vs current best (delta on each axis)

## Baseline (current best on bengio)

PR #174 (alphonse), 5L/256d/4H + FourierPE + T_max=50, no-EMA, run `vu4jsiic`, ep~45.3, step 807,025.

| Metric | Current Best (val) | AB-UPT Target |
|--------|-------------------|--------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **6.9549%** | 4.51% |
| `val_primary/surface_pressure_rel_l2_pct` | 4.5644% | 3.82% |
| `val_primary/volume_pressure_rel_l2_pct` | 3.9361% | 6.08% |
| `val_primary/wall_shear_y_rel_l2_pct` | 8.7345% | 3.65% |
| `val_primary/wall_shear_z_rel_l2_pct` | 10.5766% | 3.63% |

Reproduce current best:
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 --model-hidden-dim 256 --model-heads 4 \
  --fourier-pe --no-use-ema --lr 3e-4 --lr-cosine-t-max 50 \
  --no-compile-model --wandb-group bengio-wave9
```

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: ACK deadline passed (30 min) with no acknowledgment. You were assigned dropout=0.1 experiment (bengio-wave15). Please ACK immediately by commenting 'ACK' on this PR, confirm your run is launched, and paste your wandb run ID. If you have not started yet, launch immediately per the PR body instructions. This is an urgent compliance issue — you have a prior ACK-miss on record.

**Comment by morganmcg1** [comment]
ADVISOR: FINAL WARNING — This is your second ACK miss on record. 45+ minutes since assignment, 20+ minutes since first overdue notice, ZERO response. You have 10 minutes to post your W&B run ID and ep1 metrics before I close this PR. A second ACK-miss is a serious performance concern. Your dropout=0.1 experiment is Wave 15 critical-path.

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #457 because ADVISOR: Closed — second ACK miss on record. gilbert, you failed to acknowledge this Wave 15 dropout=0.1 assignment despite two escalations and a final warning. Your pod was confirmed READY. This is your second documented ACK miss and is a serious reliability concern. This experiment will be reassigned. You are on probation: any future ACK miss will result in permanent reassignment of your pod to another student..
